### DISCUSSION TITLE
hackintosh/updates/2021/04/24/rocket-lake

### DISCUSSION THREAD

**Discussion by giscus** [dortania/dortania.github.io#23] [Comments]
# hackintosh/updates/2021/04/24/rocket-lake

As I mentioned earlier in the monthly release notes blogpost, over the last weeks we did a great job with @vandroiy2013 in testing Intel Z590 with Comet Lake (10th generation) and Rocket Lake (11th generation) CPUs. While our experience was generally positive it was still full of unexpected caveats and nuances, so below we tried to write a summary, while our colleagues from Dortania integrate the discovered knowledge into more complete guides. All the discoveries are logically structured into sections depending on the hardware or software part. Most things do not differ from Z490, so we skipped this stuff.

https://dortania.github.io/hackintosh/updates/2021/04/24/rocket-lake.html?fbclid=IwAR2-kOA9-0aiZ4ujL5v8JsNa9lKAF69o0pJoNjLYh48xoACZ6qHxsbvS9Bk

**Comment by nguyentu98lt** [comment]
Can 500 board series support uhd 630.They don't seem to work

**Comment by LiveSorcerer** [comment]
I'm running MacOS 11.6 with OpenCore 0.7.3, on a Gigabyte Z590M Gaming X with i9 10900, but unfortunately I missed using an SSD M.2 4.0 as it's only enabled for 11gen processors. Another point that bothers me is that unfortunately we are not able to enable the fan rotation sensors, as it happens on the Z390. Other than that, it's running fine.

**Reply by SteppingSteve** [reply]
Hi, I am using a MB ASUS PRIME Z-590P, but I can't boot the PC. Could you provide me your config.plist or the EFI Folder, so I can compare it.

**Reply by joeyslaptop** [reply]
I have a Gigabyte Z490M Gaming X motherboard.  I started with a 10500.  One of the m.2 slots didn't work because of the gen10/11 issue.  I upgraded my processor to an 11600k, which enabled it.  This is great except now I have to figure out how to enable it to work with Monterey... .  Any suggestions?

**Comment by davidhyunlee** [comment]
Is Z590 a safe board pick when using a 10900K (10th gen) CPU?  My Z490 board gave out and I'm finding cheaper Z590 options online.

**Reply by pkarnadhar** [reply]
The Intel Z590 chipset is designed to work primarily with 10th and 11th generation Intel processors (Comet Lake and Rocket Lake). It's possible to use a 10th gen Intel CPU like the Core i9-10900K on a Z590 motherboard, as these motherboards often provide backward compatibility with earlier generations of CPUs.

**Comment by maxming2333** [comment]
Hello, I am using ROG MAXIMUS Z690 HERO + i9-12900k + AMD 6600 XT. However, my z690 hero uses an ALC4082 audio chip, which eventually led to the hackintosh being recognized as USB. But plugging in the audio cable can still work normally, how can I make the hackintosh system recognize it as built-in?

![image](https://user-images.githubusercontent.com/8816730/168557520-d8789731-2e8c-4242-a11b-cb2043bc6993.png)

![image](https://user-images.githubusercontent.com/8816730/168557626-ce2887a7-6d59-4d23-b934-9635c3b8b955.png)

**Reply by Boos4721** [reply]
would u share ur EFI config?

**Reply by maxming2333** [reply]
Sorry, when I solve the problem, I didn’t reply to you in time

My problem has been solved

In the end, it was found that because of ROG Maximus Z690 Hero, in order to improve the audio quality, it used the USB mode output

It is mentioned in ROG MAXIMUS Z690 Hero's official website

However, I always thought that ROG Maximus Z690 Hero was a board chip like other motherboards, so a large number of USB Audio appeared to make me doubt

Now the problem has been solved, because the motherboard is originally used to output audio, so MacOS recognizes it as USB Audio

This motherboard is so strange

**Reply by maxming2333** [reply]
https://rog.asus.com/motherboards/rog-maximus/rog-maximus-z690-hero-model/

"The ALC4082 codec uses a USB interface instead of the conventional high definition audio (HDA) interface and improves playback audio resolution from 192 to 384 kHz."
![image](https://user-images.githubusercontent.com/8816730/197532839-a6e80d1d-2053-4554-92d3-388410286539.png)
